.github/ISSUE_TEMPLATE/docs-report--quasar-v2.yml

```
@@ -8,18 +8,17 @@ body:
     attributes:
       value: |
         ### Read Before Proceeding Further
+        This form is for submitting a **docs report** for Quasar Framework(**v2**). (https://quasar.dev).
```

```suggestion

        This form is for submitting a **docs report** for Quasar Framework(**v2**). (https://quasar.dev)
```
the empty line is missing.